components/MetadataCard.vue

```diff
@@ -57,6 +59,7 @@ export default class MetadataCard extends Vue {
   @Prop(Object) readonly allExif!: Object
 
   exifInfo: Object = {}
+  isLoading: boolean = true
```

Comment:
I suggest renaming this to more specify `hasLoadedImage`, with default is `false`